src/beacon/tests/configuration/test_environment_variables.py

```
@@ -0,0 +1,78 @@
+import os
+import pytest
```

Ruff (0.11.9)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)